### PR TITLE
added the bash commands in pr_check.sh script

### PR DESCRIPTION
**Pull request by vinaybommana**
Hi @deepak1725 please check the script and suggest me if I am missing something

### CODE REVIEW

**Comment by deepak1725** [comment]
I suggest, Running 3 Tests in `pr_check.sh`
- Unit Tests
- Linting Tests (Code Linting + `openshift/template.yaml` listing)
- Integration Tests

It's fine if you want to introduce separate Jobs for each of them. 
The whole idea is to make the New PR's not able to break the Application.  

Reference: https://github.com/fabric8-analytics/fabric8-analytics-server/pull/821 Check the Number of Tests that run against the code.

**Comment by deepak1725** [comment]
That's much better. Thanks.
Few things:
- Please break Unit Tests and Code Linting into 2 separate Jobs.
- Please do add another job for openshift template linting. 

For Openshift Linting, you can take references from earlier shared PR.

**Comment by deepak1725** [comment]
Do keep an eye on having an empty line at the end of the file. (Check the diff)

**Comment by deepak1725** [comment]
> sure, will do that.. also can you please suggest me the node version we should be using, I've checked the version in the glitchtip-frontend and it is defined as 16, so can i use that for the github action as well.. should we be testing for older versions of node as well ?

Fine with Node 16 as of now.

File: .github/workflows/pr_checks.yml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+          git clone https://github.com/rh-cssre/glitchtip-backend.git
+          cd glitchtip-backend;
+          docker-compose run --rm web ./manage.py migrate
+          docker-compose up -d
+      # start glitchtip-frontend server
+      - run: sleep 150
+      - run: npm ci
+      - run: npm start & npx wait-on http://localhost:4200
+      - run: sleep 30
+      - run: npm run cy:run
```
Comment (deepak1725) [inline .github/workflows/pr_checks.yml:81]: Do you want to run `cy:open` or `cy:run` ?

File: .github/workflows/pr_checks.yml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+      
+      - name: start glitchtip-backend server
+        run: |
+          # clone glitchtip-backend repo
+          git clone https://github.com/rh-cssre/glitchtip-backend.git
+          cd glitchtip-backend;
+          docker-compose run --rm web ./manage.py migrate
+          docker-compose up -d
+      # start glitchtip-frontend server
+      - run: sleep 150
```
Comment (deepak1725) [inline .github/workflows/pr_checks.yml:77]: Can we add a Ping command if, after 150 seconds, Container actually running up?

**Review by deepak1725 — CHANGES_REQUESTED** [review]
Looks fine, 
Do add an Empty line in py_check.sh

Minor nits gits

File: .github/workflows/pr_checks.yml (hunk 1 above)
Comment (vinaybommana) [reply .github/workflows/pr_checks.yml:81]: Hi @deepak1725,
> Do you want to run `cy:open` or `cy:run` ?

`cy:open` is for development testing the cypress in browser like it opens up a chrome window and is used for interactive
that is why I've used `cy:run`

File: .github/workflows/pr_checks.yml (hunk 2 above)
Comment (vinaybommana) [reply .github/workflows/pr_checks.yml:77]: sure, let me do that

**Review by vinaybommana — COMMENTED** [review]
this is done

**Review by deepak1725 — APPROVED** [review]
Looks Good. Thanks

**Pull request merged**